.github/workflows/gradle.yml

```diff
@@ -9,31 +9,22 @@ on:
 
 jobs:
   build:
-    strategy:
-      matrix:
-        java: [1.8, 11]
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
-    env:
-      gradle_version: 6.8.3 # set to empty to build with most recent version of gradle
-      gradle_commands: build # default is build
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK ${{ matrix.java }}
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java }}
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
-        with:
-          path: ~/.gradle/caches
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-          restore-keys: ${{ runner.os }}-gradle
-      - name: Wrap with specified version
-        run: gradle wrapper --gradle-version=${{ env.gradle_version }}
-        if: ${{ env.gradle_version != '' }}
-      - name: Wrap without version
-        run: gradle wrapper
-        if: ${{ env.gradle_version == '' }}
-      - name: Run commands
-        run: ./gradlew ${{ env.gradle_commands }}
+    uses: ome/action-workflows/.github/workflows/gradle_build.yml@v1
+  publish_snapshots:
```

Comment:
At first glance it looks like it should be possible to merge `publish_snapshots` and `publish` as the only varying component is the URL of the target repository.

Reply:
I kept them separate for clarity

Reply:
Also since it is 2 different jobs, passing variable between jobs is tricky and usually involved a tmp file